packages/@okta/vuepress-site/docs/concepts/okta-data-model/index.md

```diff
@@ -14,20 +14,20 @@ Entities inside Okta are referred to as resources. Each Okta resource contains:
+A top-level directory resource is available for your user base in your org. You can host directories either inside Okta or replicate outside user directories, as in the case of Active Directory or LDAP.
 
-Your end users are modeled inside Okta as [**Users**](/docs/reference/api/users/). Users are associated with an email address and a username. The username must be unique, but it can be changed (and often is when, for example, someone changes their name). The email address doesn't need to be unique and often isn't. Uniqueness for a user is determined by their Okta user ID.
+Your end users are modeled inside Okta as [**Users**](/docs/reference/api/users/). Users are associated with an email address and a username. The username must be unique, but it can be changed (and often is when, for example, someone changes their name). The email address doesn't need to be unique and often isn't. The Okta user ID determines the uniqueness for a user.
 
-A [**Group**](/docs/reference/api/groups/) is made up of users found within a directory. It can be thought of as a label applied to a set of users. A user can be a member of multiple groups. Groups are useful for representing roles, relationships, and can even be used for subscription tiers.
+A [**Group**](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Group/) is made up of users found within a directory. It can be thought of as a label applied to a set of users. A user can be a member of multiple groups. Groups are useful for representing roles, relationships, and can even be used for subscription tiers.
 
-An [**Application**](/docs/reference/api/apps/) holds information about the protocol in which it wants Okta to communicate, policies for accessing the application, and which users can use the application after identifying themselves.
+An [**Application**](/docs/reference/api/apps/) holds information about the protocol in which it wants Okta to communicate, policies for accessing the app and which users can use the app after identifying themselves.
 
-The relationship between an application and a user is stored in Okta as an [AppUser](/docs/reference/api/apps/#assign-user-to-application-for-sso). This allows for **mappings** between profile values where they are different. For example, application1 may refer to a user's last name as `lastName` and application2 may refer to last name as `surname`. The AppUser stores the value as necessary for the application.
+The relationship between an app and a user is stored in Okta as an [AppUser](/docs/reference/api/apps/#assign-user-to-application-for-sso). This allows for **mappings** between profile values where they’re different. For example, app1 may refer to a user's last name as `lastName` and app2 may refer to last name as `surname`. The AppUser stores the value as necessary for the app.
```

Comment:
Do we usually use bold to highlight terms? I thought it was just for ui elements.

Reply:
Not sure. This was in there before, but happy to take it out.

Reply:
Just double checked if there was anything funky in the devdocs version of the style guide to allow this, and I don't see it. I'll remove them, good catch!